shared/features/invoice/tokens/invoice-tokens.ts

```diff
@@ -0,0 +1,65 @@
+export const InvoiceTokens = {
+  header: {
+    sampleTitle: "Your invoice number",
+    receiptTitle: "Receipt NO:",
+    invoiceTitle: "Invoice NO:",
+  },
+  invoiceInfos: {
+    wagmiName: "Wagmi",
+    wagmiAddress: "54 Rue Du Faubourg Montmartre, Paris, France, 75009",
```

Comment:
Check with Camille but the address has changed https://onlydust.slack.com/archives/C031KBA3TJR/p1731405785428079